cmd/ipfs-cluster-ctl/formatters.go

```diff
@@ -208,6 +210,37 @@ func textFormatPrintMetric(obj *api.Metric) {
 	fmt.Printf("%s: %s | Expire: %s\n", peer.IDB58Encode(obj.Peer), obj.Value, date)
 }
 
+func textFormatPrintGlobaleRepoGC(obj *api.GlobalRepoGC) {
+	peers := make(sort.StringSlice, 0, len(obj.PeerMap))
+	for peer := range obj.PeerMap {
+		peers = append(peers, peer)
+	}
+	peers.Sort()
+
+	for _, peer := range peers {
+		item := obj.PeerMap[peer]
+		// If peer name is set, use it instead of peer ID.
+		if len(item.Peername) > 0 {
+			peer = item.Peername
+		}
+		if item.Error != "" {
+			fmt.Printf("%-15s | ERROR: %s\n", peer, item.Error)
+		} else {
+			fmt.Printf("%-15s\n", peer)
+		}
+
+		fmt.Printf("  > CIDs:\n")
+		for _, key := range item.Keys {
+			if key.Error != "" {
+				fmt.Printf("    - %s | ERROR: %s\n", key.Key.String(), key.Error)
```

Comment:
Unfortunately ipfs will not set the `Key` when an error happens, it will make it part of the error message (next time please look things like this up before deciding on an approach).